servers/rendering/rendering_device_driver.h

```diff
@@ -190,6 +192,10 @@ class RenderingDeviceDriver : public RenderingDeviceCommons {
 		BUFFER_USAGE_INDEX_BIT = (1 << 6),
 		BUFFER_USAGE_VERTEX_BIT = (1 << 7),
 		BUFFER_USAGE_INDIRECT_BIT = (1 << 8),
+		BUFFER_USAGE_SHADER_BINDING_TABLE_BIT = (1 << 10),
+		BUFFER_USAGE_SHADER_DEVICE_ADDRESS_BIT = 0x00020000,
+		BUFFER_USAGE_ACCELERATION_STRUCTURE_BUILD_INPUT_READ_ONLY_BIT = 0x00080000,
+		BUFFER_USAGE_ACCELERATION_STRUCTURE_STORAGE_BIT = 0x00100000,
```

Comment:
Out of curiosity, why are these written as hexadecimal numbers instead of bitshifts? (Same for the ones in the enums further below.)

Reply:
IIRC I copy-pasted these from the Vulkan header :D
Let's use the bitshifts-style.